.github/workflows/test-forked.yml

```diff
@@ -19,9 +29,9 @@ jobs:
       - name: allowed message
         run: echo "Allowed to run"
 
-  unit-tests-forked:
+  int-tests:
     needs: allowed-forked
-    uses: ./.github/workflows/test-unit.yml
+    uses: ./.github/workflows/integration-test.yml
```

Comment:
I wonder if this makes workflow file `./.github/workflows/integration-test-forked.yml` unnecessary now.